scripts/mypuush

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+##############################################################################
+###                                                                        ###
+###  File : mypuush                                                        ###
+###  Language : bash                                                       ###
+###  Description : takes a screenshot of your screen, uploads the file     ###
+###     and send the link it the clipboard                                 ###
+###                                                                        ###
+###  Dependencies : xclip, jq, gnome-screenshot                            ###
+###                                                                        ###
+###  Author : Raphael Ehret                                                ###
+###  E-mail : raphael.ehret@gmail.com                                      ###
+###                                                                        ###
+##############################################################################
+
+
+print_help () {
+
+    name=$(basename $0)
+
+    echo "$name takes a screenshot, uploads it and sends the link in your clipboard"
+    echo "Usage : $name [option] [-a|-w]"
+    echo ""
+    echo "$name [option] -a"
```

Comment:
Or, considering your implementation, you can say that every arguments are forwarded to gnome-screenshot. But that's not really good (code injection).